<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Demo_RoundProgressBar</class>
 <widget class="QDialog" name="Demo_RoundProgressBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Donut Style</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Pie Style</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Line Style</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRoundProgressBar" name="RoundBar1" native="true"/>
     </item>
     <item row="1" column="1">
      <widget class="QRoundProgressBar" name="RoundBar2" native="true"/>
     </item>
     <item row="1" column="2">
      <widget class="QRoundProgressBar" name="RoundBar3" native="true"/>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QSlider" name="Slider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRoundProgressBar" name="RoundBar4" native="true"/>
     </item>
     <item row="3" column="1">
      <widget class="QRoundProgressBar" name="RoundBar5" native="true"/>
     </item>
     <item row="3" column="2">
      <widget class="QRoundProgressBar" name="RoundBar6" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Donut Style (dark styled)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRoundProgressBar" name="RoundBar7" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QRoundProgressBar</class>
   <extends>QWidget</extends>
   <header>qroundprogressbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
